google/cloud/aiplatform/metadata/schema/system_schema.py

```diff
@@ -0,0 +1,508 @@
+        """
+        extended_metadata = metadata or {}
+        extended_metadata["accuracy"] = accuracy
+        extended_metadata["precision"] = precision
+        extended_metadata["recall"] = recall
+        extended_metadata["f1score"] = f1score
+        extended_metadata["mean_absolute_error"] = mean_absolute_error
+        extended_metadata["mean_squared_error"] = mean_squared_error
+        extended_metadata["resourceName"] = metrics_name
+
+        super(Metrics, self).__init__(
+            schema_title=self.SCHEMA_TITLE,
+            resource_name=metrics_name,
+            uri=uri,
+            display_name=display_name,
+            schema_version=schema_version,
+            description=description,
+            metadata=extended_metadata,
+            state=state,
+            kwargs=kwargs,
+        )
+
+
+class ContainerExecution(base_execution.BaseExecutionSchema):
+    """Execution schema for a container execution."""
+
+    SCHEMA_TITLE = "system.ContainerExecution"
+
+    def __init__(
+        self,
+        state: gca_execution.Execution.State = gca_execution.Execution.State.RUNNING,
+        execution_name: Optional[str] = None,
+        display_name: Optional[str] = None,
+        schema_version: Optional[str] = None,
+        metadata: Optional[Dict] = None,
+        description: Optional[str] = None,
+        **kwargs,
+    ):
+        """Args:
+        state (gca_execution.Execution.State.RUNNING):
+            Optional. State of this Execution. Defaults to RUNNING.
+        execution_name (str):
+            Optional. The resource name of the Execution following the format as follows.
+            This is globally unique in a metadataStore:
+            projects/123/locations/us-central1/metadataStores/<metadata_store_id>/executions/<resource_id>.
+        display_name (str):
+            Optional. The user-defined name of the Execution.
+        schema_version (str):
+            Optional. schema_version specifies the version used by the Execution.
+            If not set, defaults to use the latest version.
+        metadata (Dict):
+            Optional. Contains the metadata information that will be stored in the Execution.
+        description (str):
+            Optional. Describes the purpose of the Execution to be created.
+        **kwargs:
+            Optional. Additional Args that will be passed directly to the Execution base method for backward compatibility.
+        """
+        extended_metadata = metadata or {}
+        extended_metadata["resourceName"] = execution_name
+        super(ContainerExecution, self).__init__(
+            schema_title=self.SCHEMA_TITLE,
+            resource_name=execution_name,
+            state=state,
+            display_name=display_name,
+            schema_version=schema_version,
+            description=description,
+            metadata=extended_metadata,
+            kwargs=kwargs,
+        )
+
+
+class ImporterExecution(base_execution.BaseExecutionSchema):
```

Comment:
What's the use case for including Importer/Resolver/Dag execution, they all seem KFP/Pipelines specific?

Reply:
That is correct, however since they are not the main target for this initial release removing all three.